setup-ubuntu-16.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Fail if command failed, variable is unassigned, also show executed commands
+set -eux
+
+sudo apt-get update -qq
+
+echo "Install Boehm GC, libunwind, RE2"
+sudo apt-get install -y -qq \
+  clang++-3.9 \
+  libgc-dev \
+  libunwind8-dev \
+  libre2-dev
+
+echo "Install argonaut from source"
+rm -rf argonaut
+git clone https://github.com/argonaut-io/argonaut.git
```

Comment:
Can we hardcode a commit here so that builds are reproducible? i.e. `git checkout <rev>`

Reply:
Good idea - this is now done against latest Argonaut `master` at the time of writing: https://github.com/argonaut-io/argonaut/commit/2c719f155744881d30fc932dcbbf597a9ce8084c